repo/packages/G/geoserver/0/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion" : "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name" : "geoserver",
+  "version" : "2.10.0-1.1.0",
```

Comment:
Universe packaging version here should be 0.0.1. 
So, `version` would be `geoserverVersion-packagingVersion`